hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
```

Comment:
but adding the grep to diff makes it the ones from the diff in
integration-cli only

On Wed, Apr 22, 2015 at 1:31 PM, Doug Davis <notifications@github.com>
wrote:

> In hack/make/validate-test
> <https://github.com/docker/docker/pull/12646#discussion_r28911708>:
>
> > @@ -0,0 +1,34 @@
> > +#!/bin/bash
> > +
> > +# Make sure we're not using gos' Testing package any more in integration-cli
> > +
> > +source "$(dirname "$BASH_SOURCE")/.validate"
> > +
> > +IFS=$'\n'
> > +files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
>
> yes if we want to check all files instead of just the ones changed by the
> current PR.
> I was following the pattern set by the other validate-* scripts. If you're
> ok with the inconsistency I'll change it.
> Just let me know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12646/files#r28911708>.
>